yarn-project/noir-contracts/bootstrap.sh

```diff
@@ -1,15 +1,3 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-# Install noir if it is not installed already
-if ! command -v noirup &> /dev/null
-then
-    echo "Installing noir"
-    source ./scripts/install_noirup.sh
-fi
-
-# Update noir
-./scripts/install_noir.sh
-
-# Use yarn script to compile and create types
-yarn 
```

Comment:
removed this because the workspace will have installed dependencies before calling bootstrap in here.